### PR TITLE
Port some commands to Windows

### PR DESCRIPTION
**Pull request by duester**
Partly implements https://github.com/reibitto/command-center/issues/80:
- ExternalIPCommand
- LockCommand
- TimerCommand
- ToggleHiddenFilesCommand

### CODE REVIEW

**Comment by duester** [comment]
That's how the pop-up timer notification looks like:

File: core/src/main/scala/commandcenter/command/ExternalIPCommand.scala [hunk 1]
```diff
@@ -11,19 +11,22 @@ final case class ExternalIPCommand(commandNames: List[String]) extends Command[S
...
                       .bimap(CommandError.UnexpectedException, _.trim)
     } yield List(
       Preview(externalIP)
         .score(Scores.high(input.context))
         .onRun(tools.setClipboard(externalIP))
     )
+
+  private def pCommand: PCommand =
+    OS.os match {
+      case OS.Windows => PCommand("powershell", "(Invoke-WebRequest -uri \"api.ipify.org\").Content")
```
Comment (reibitto) [inline core/src/main/scala/commandcenter/command/ExternalIPCommand.scala:27]: This works, but I think going down a level to Powershell is better suited for cases that have no other easy alternative.

I'd probably change this method to be `getExternalIP` and use `Sttp` to make the http request for the Windows case while keeping `PCommand` for the others.

I figure this should be faster too. I imagine starting up Powershell has some additional latency.

**Review by reibitto — CHANGES_REQUESTED** [review]
Thanks for doing this. I appreciate it. 🙂

Powershell and Registry. Funky. 😛 Still better than AppleScript though!

Anyway, looks great. Only one minor thing.

File: core/src/main/scala/commandcenter/command/ExternalIPCommand.scala (hunk 1 above)
Comment (duester) [reply core/src/main/scala/commandcenter/command/ExternalIPCommand.scala:27]: Ok, fair point. Thanks, it's really funky and was an interesting experience to me to learn to work with Powershell as I had never done it before.

**Review by reibitto — APPROVED** [review]
Wonderful. Thanks!

**Pull request merged**